'''
    Tratamento de erros
'''


# try:
#     a = 10
#     b = 0
#     c = a / b

# except ZeroDivisionError as error:
#     print('Não é possível fazer a divisão por 0.')

# except NameError as error:
#     print('Umas das variávees não está definida')

# finally -> É um bloco que sempre será executado no final do código


try:
    ...

finally:
    print('Sempre será executado.')

























